crates/re_types_builder/src/codegen/rust/serializer.rs

```diff
@@ -476,6 +492,9 @@ fn quote_arrow_field_serializer(
         DataType::List(inner) | DataType::FixedSizeList(inner, _) => {
             let inner_datatype = inner.data_type();
 
+            let expose_inner_as_buffer = is_backed_by_arrow_buffer(inner.data_type())
+                && matches!(datatype, DataType::List(_));
```

Comment:
We should be able to optimize `FixedSizeList` too in the future, correct (but probably not in the same way)?

Reply:
That said, if we find a case where we have a *large* fixed-sized buffer we can add an attribute to force this instead.